src/main/java/org/broadinstitute/dsm/DSMServer.java

```diff
@@ -294,10 +306,51 @@ protected void setupCustomRouting(@NonNull Config cfg) {
         patch(UI_ROOT + RoutePath.USER_SETTINGS_REQUEST, new UserSettingRoute(), new JsonTransformer());
 
         setupJobs(cfg, kitUtil, notificationUtil, eventUtil, container, receiver);
+        
+        setupPubSub();
 
         logger.info("Finished setting up DSM custom routes and jobs...");
     }
 
+    private void setupPubSub() {
+        String projectId = "hl7-dev-274911";
+        String subscriptionId = "pegah-results-sub";
+
+        try {
+            ProjectSubscriptionName subscriptionName =
+                    ProjectSubscriptionName.of(projectId, subscriptionId);
+
+            // Instantiate an asynchronous message receiver.
+            MessageReceiver receiver =
+                    (PubsubMessage message, AckReplyConsumer consumer) -> {
+                        // Handle incoming message, then ack the received message.
+                        System.out.println("Id: " + message.getMessageId());
+                        System.out.println("Data: " + message.getData().toStringUtf8());
```

Comment:
could we change the system.out to logger.info please

Reply:
I'll remove them after we test today anyways